sequencing-server/jestGlobalSetup.js

```diff
@@ -4,7 +4,8 @@ config();
 
 process.env.MERLIN_GATEWAY_URL = process.env.MERLIN_GATEWAY_URL ?? 'http://localhost:9000';
 process.env.MERLIN_GRAPHQL_URL = process.env.MERLIN_GRAPHQL_URL ?? 'http://localhost:8080/v1/graphql';
-process.env.HASURA_GRAPHQL_ADMIN_SECRET = process.env.HASURA_GRAPHQL_ADMIN_SECRET ?? '';
+process.env.HASURA_GRAPHQL_JWT_SECRET = process.env.HASURA_GRAPHQL_JWT_SECRET ?? '{ "type": "HS256", "key": "oursupersecretsupersecurekey1234567890" }'
```

Comment:
We should not pass the `HASURA_GRAPHQL_JWT_SECRET` to the sequencing server since as a service the sequencing server should not be allowed to mint authentication tokens. Using the `HASURA_GRAPHQL_ADMIN_SECRET` should be enough since (for now) it gives this service the highest privilege  to query Hasrua.